### DISCUSSION TITLE
Usage analytics and Data privacy

### DISCUSSION THREAD

**Discussion by SlavaKeshkov** [microsoft/PubSec-Info-Assistant#705] [Q&A]
Hi Microsoft team,

Thanks for releasing the accelerator!

Two questions on my side:
1. Does the assistant support user analytics: How can the development team view how many users used the assistant based on the Azure logs?
2. Does the assistant log the conversations that are done with it or they are not logged? In case they are, is there any way for the user to opt-out of saving chat history?

Thanks a lot in advance!

**Comment by dayland** [comment]
We do not collect user logins/use of the app nor chat/conversation history. These are not available in any of the logs.
